playbook.md

```diff
@@ -69,23 +69,29 @@ If you wish to change the config in `01_defaults.env` or `03_backend.env`, you
 need to merge a change to the `services/jobrunner/defaults.env` or
 `BACKEND/backend.env`, and update the infrastructure code as above.
 
+
 ### Deploy job-runner
 
-In a clone of the [backend-server](https://github.com/opensafely-core/backend-server) repository, run:
+1. If there dependency updates, update lib
 
 ```bash
-just update-jobrunner
+just jobrunner/update-dependencies
```

Comment:
should we just automatically run this every time?

Reply:
Not a bad idea.  Means that landing changes to it need to have a corresponding job-server commit without delay.

Reply:
hmm, yeah, ideally this would work like other dependency management - update jobrunner & then install this at the correct pinned version, but that's starting to sound like its own mini-project. (it also sounds like re-inventing the wheel, but that's another matter, due to -constraints- we may need to...?)

The thing that made me query this was "if there are dependency updates" - how does the operator know? this happens before the update script outputs the list of commits to be deployed for inspection - I think either it runs automatically or the workflow needs a slight rethink?

Reply:
It think this is addressed in the latest commit.

Essentially, we integrates the dependency update into the deploy script.

This is not ideal, as it *could* lead to breakage if things get out of sync. But that's easily fixable.